extensions/parquet/compression/README.md

```diff
@@ -0,0 +1,30 @@
+#Parquet compression support
```

Comment:
I'm pretty sure you need a space for this to properly be a heading. IntelliJ's renderer seems to agree.
```suggestion
# Parquet compression support
```